### DISCUSSION TITLE
Change feed's unread count position in sidebar for readability

### DISCUSSION THREAD

**Discussion by Grishkaone** [FreshRSS/FreshRSS#4068] [Ideas]
Hi everyone 👋

I'm using FreshRSS since a few days, with great pleasure. 
There is one point in the interface that I find difficult to read, in the sidebar, because to different information are not separated enough.

Example :
![old](https://user-images.githubusercontent.com/46840243/146190921-0711652d-a7b2-4feb-a8a0-b5c812868ec3.png)

For the category, the unread count is align on the left of the name and it's very clear.
But for each feed, the unread count is align on the right of the name : there is little space and the count is not immediately readable. I know there is unread stuff because the name is bold (a good thing).

Now, I modified a little the template.css file (lines 1191) :
![new](https://user-images.githubusercontent.com/46840243/146191405-8d57e926-21e1-41f0-b8c8-5c73ce1416b2.png)
* The unread count is align on the right, which is consistent with the one for category.
* There is enough space for better readability.

What do you think ?
I'm not a CSS wizard, so there is certainly room for improvement.
The original code :
`.feed .item-title:not([data-unread="0"])::before {
	content: "(" attr(data-unread) ") ";
}`

The new code :
`.feed .item-title:not([data-unread="0"])::after {
	content: "(" attr(data-unread) ") ";
	position: absolute;
	right : 15px;
}`

I don't know which section is best "idea" or "show and tell", but here it is.

Have a nice day 🖖

**Comment by Grishkaone** [comment]
OK, I can see a first issue with this solution : if the feed's name is long enough, the unread count is superimposed on the text 🤔.

**Reply by Frenzie** [reply]
I'd say that unless you add alternating background colors, that makes it a little harder to read. And that these kinds of small personal mods are often exactly what CustomCSS is for. :-)

**Comment by Grishkaone** [comment]
I didn't notice the CustomCSS extension yet, thanks @Frenzie !
I'm trying more styling with space and another symbol than the "( )". The `position: absolute;` was not good when scrolling. And the left position is something mixed with long feed's title.

**Reply by Frenzie** [reply]
You'll probably be able to do anything you like by going full `display: grid` but it might be a bit complex to wrap your head around at first if you're not familiar with it yet.

**Comment by math-GH** [comment]
Try this for theme "dark":

```
.tree-folder-items .item.feed {
	position: relative;
}
.feed .item-title:not([data-unread="0"])::before{
	content: "";
}
.feed .item-title:not([data-unread="0"])::after {
	content: "(" attr(data-unread) ") ";
	position: absolute;
	right: 10px; 
	background: #161616;
	padding: 0 0 0 5px;
}
```

**Reply by Grishkaone** [reply]
Wow, thank you @math-GH !

Excellent solution, it works great 👍.
I just changed the background value to `transparent` so it work also with "Ansum" and "Dark" themes.
I really love this layout 😍.

**Reply by math-GH** [reply]
You will need the background for long feed names.

It is also a great way to make it more beautiful, f.e. with rounded corners.

**Reply by math-GH** [reply]
@Frenzie @Alkarex : Should we improve the UI to align the unread numbers on the feed level like the category level?

**Reply by Grishkaone** [reply]
> You will need the background for long feed names.
> 
> It is also a great way to make it more beautiful, f.e. with rounded corners.

Hey !

Indeed. For know, I use only short name and I've changed a value with Custom Css to limite the available length for feed's title :
```
.tree-folder-items > .item > a {
width: calc(100% - 25%);
```
![image](https://user-images.githubusercontent.com/46840243/147939983-cef96d0a-214b-4fa2-945f-d39f2c412caa.png)

About the initial unread count that is placed at the beginning of the line, I have to complete a little. Now it works with others theme, like Ansum :
```
.tree .tree-folder .tree-folder-items .feed .item-title:not([data-unread="0"])::before{
	content: "";
	display: none;
}
```

Without it, I had two unread count :
![image](https://user-images.githubusercontent.com/46840243/147939755-1cbc4ab8-5aa8-47a8-9376-370b57b34ce4.png)

**Reply by Alkarex** [reply]
> @Alkarex : Should we improve the UI to align the unread numbers on the feed level like the category level?

@math-GH  Sorry for the delay. I do not really mind. It could indeed be more consistent

**Comment by math-GH** [comment]
Created a pull request to make it default: https://github.com/FreshRSS/FreshRSS/pull/4166
